### ISSUE TITLE
Increase parameter synced data limit beyond 256 bits

### ISSUE THREAD

**Issue by EnderShadow** [vrchat-community/osc#163]
*What's the idea?**
256 bits is too few bits especially when combined when something like the DiverX ContactGlove which requires 107 or 177 bits to show finger position on your avatar. The amount of bits required by it reduces the usable synced parameter amount to either 149 or 79 bits which brings us to the increase request from 2 years ago #15.

**Is there another way?**
Alternative solutions to increasing the number of synced bits include
- ints and floats which are less than 8 bits (not sure if this is possible in Unity)
- an alternative way to get hand tracking through other devices similar to how the quest can do it.

**Additional context**
I think increasing it to 512 bits should probably be good like how it was suggested in the above linked issue, but I'm not sure.

**Comment by cadergator10** [comment]
Very late to the party but I second this. Face tracking takes a whole 160 bits, leaving no room for anything else

**Comment by purple2428** [comment]
Completely agreed, if you want to have any assets added to your avatar you basically cannot - especially if you use a high-quality avatar base with a lot of features: ex, the Vulper. 512 is a bare minimum now with the 160 bits required for face tracking. 768 would be ideal.

**Comment by MiguVT** [comment]
It was soo much years old this issue and still not fixed, hope it get fixed soon cause now too much people use face tracking + SPS and with that + Dynamic Expressions and our own parameters its impossible to keep all as we wanted.

Edit: Its also deprecated, maybe refactor the network protocol to have a more efficient network protocol allowing more parameters. The main problem is that VRChat network protocol needs a rewrite, without a rewrite increase it to 512 will make everything harder to mantain stability.

**Comment by Kiiinji** [comment]
Absolutely mandatory today with Face Tracking, it's baffling that we're still locked with 256 bits in 2026..

**Comment by MiguVT** [comment]
> Absolutely mandatory today with Face Tracking, it's baffling that we're still locked with 256 bits in 2026..

Yep, there is lot of things that is allowed and is really poorly optimized but hey 256bits are the max like noway
